R/utilities-units.R

```diff
@@ -44,15 +45,21 @@ toBaseUnit <- function(quantity, values, unit) {
 #'
 #' valuesInMl <- toUnit(par, c(1, 5, 5), "ml")
 #' @export
-toUnit <- function(quantity, values, targetUnit) {
-  validateIsOfType(quantity, Quantity)
+toUnit <- function(quantityOrDimension, values, targetUnit, molWeight = NULL) {

```

Comment:
@pchelle This is the method to convert values in any type of unit

for example

`toUnit("Mass", c(1, 2, 3), "mol", 20)` will assume that you want to convert the array (1, 2,3) from the base unit of dimension Mass in mol by using the molweight value

We can discuss if this is the apprpriate interface once I see how you need to use it. Please feedback